server/src/test/java/org/opensearch/search/query/QueryPhaseTests.java

```diff
@@ -114,13 +106,8 @@
 import java.util.Collections;
 import java.util.List;
 
-import static org.opensearch.search.query.QueryPhase.pointsHaveDuplicateData;
+import static org.hamcrest.Matchers.*;
```

Comment:
oopsie wildcard insert

Reply:
Thank you for catching it, I will fix this